### PR TITLE
Consolidate all YMDK keyboards into the YMDK directory

### PR DESCRIPTION
**Pull request by ennation**
## Description

* Consolidate all YMDK keyboards to the YMDK directory, as some were in it and some were not. The YD60MQ, YMD75, YMD96, and YMD_NP21 have been moved into the directory
* Update example make commands in the readmes of all YMDK keyboards to use the newer `qmk` script, and to reflect the movement of the moved keyboards

The exception is the YD68, as although it is named similarly to the YD60, I cannot find anything definitely tying it to YMDK, so I left it out as a precaution.

*Currently resolving YMD75 build issues resulting from the move.*

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ymdk/yd60mq/readme.md [hunk 1]
```diff
@@ -13,17 +13,17 @@ Customizable 60% PCB by [YMDK](https://ymdk.aliexpress.com/store/429151?spm=2114
 Make example for this keyboard (after setting up your build environment):
 
 ```sh
-make yd60mq:default        # builds the 12 LED variant
-make yd60mq/12led:default  # also builds the 12 LED variant
-make yd60mq/16led:default  # builds the 16 LED variant
+make ymdk/yd60mq:default        # builds the 12 LED variant
+make ymdk/yd60mq/12led:default  # also builds the 12 LED variant
+make ymdk/yd60mq/16led:default  # builds the 16 LED variant
```
Comment (filterpaper) [inline keyboards/ymdk/yd60mq/readme.md:18]:
```suggestion
qmk compile -kb ymdk/yd60mq -km default        # builds the 12 LED variant
qmk compile -kb ymdk/yd60mq/12led -km default  # also builds the 12 LED variant
qmk compile -kb ymdk/yd60mq/16led -km default  # builds the 16 LED variant
```

File: keyboards/ymdk/yd60mq/readme.md [hunk 2]
```diff
@@ -13,17 +13,17 @@ Customizable 60% PCB by [YMDK](https://ymdk.aliexpress.com/store/429151?spm=2114
...
 
 The keyboard uses a DFU bootloader. To make a keymap and use dfu to flash it:
 
 ```sh
-make yd60mq:default:flash        # builds and flashes the 12 LED variant
-make yd60mq/12led:default:flash  # also builds and flashes the 12 LED variant
-make yd60mq/16led:default:flash  # builds and flashes the 16 LED variant
+make ymdk/yd60mq:default:flash        # builds and flashes the 12 LED variant
+make ymdk/yd60mq/12led:default:flash  # also builds and flashes the 12 LED variant
+make ymdk/yd60mq/16led:default:flash  # builds and flashes the 16 LED variant
```
Comment (filterpaper) [inline keyboards/ymdk/yd60mq/readme.md:26]:
```suggestion
qmk flash -kb ymdk/yd60mq -km default        # builds and flashes the 12 LED variant
qmk flash -kb ymdk/yd60mq/12led -km default  # also builds and flashes the 12 LED variant
qmk flash -kb ymdk/yd60mq/16led -km default  # builds and flashes the 16 LED variant
```

File: keyboards/ymdk/ymd75/readme.md [hunk 3]
```diff
@@ -13,11 +13,11 @@ This firmware can also be flashed onto the KBDFans KBD75 Round 2 Bootmapper Clie
 
 Make example for this keyboard (after setting up your build environment):
 
-    make ymd75/rev1:default
+    make ymdk/ymd75/rev1:default
```
Comment (filterpaper) [inline keyboards/ymdk/ymd75/readme.md:16]:
```suggestion
    qmk compile -kb ymdk/ymd75/rev1 -km default
```

File: keyboards/ymdk/ymd75/readme.md [hunk 4]
```diff
@@ -13,11 +13,11 @@ This firmware can also be flashed onto the KBDFans KBD75 Round 2 Bootmapper Clie
...
 
 Flashing example for this keyboard ([after setting up the bootloadHID flashing environment](https://docs.qmk.fm/#/flashing_bootloadhid))
 
-    make ymd75/rev1:default:flash
+    make ymdk/ymd75/rev1:default:flash
```
Comment (filterpaper) [inline keyboards/ymdk/ymd75/readme.md:20]:
```suggestion
    qmk flash -kb ymdk/ymd75/rev1 -km default
```

File: keyboards/ymdk/ymd96/readme.md [hunk 5]
```diff
@@ -8,11 +8,11 @@ A 96-keyboard which supports both ANSI and ISO along with many different layout
 
 Make example for this keyboard (after setting up your build environment):
 
-    make ymd96:default
+    make ymdk/ymd96:default
```
Comment (filterpaper) [inline keyboards/ymdk/ymd96/readme.md:11]:
```suggestion
    qmk compile -kb ymdk/ymd96 -km default
```

File: keyboards/ymdk/ymd96/readme.md [hunk 6]
```diff
@@ -8,11 +8,11 @@ A 96-keyboard which supports both ANSI and ISO along with many different layout
...
 
 Flashing example for this keyboard ([after setting up the bootloadHID flashing environment](https://docs.qmk.fm/#/flashing_bootloadhid))
 
-    make ymd96:default:flash
+    make ymdk/ymd96:default:flash
```
Comment (filterpaper) [inline keyboards/ymdk/ymd96/readme.md:15]:
```suggestion
    qmk flash -kb ymdk/ymd96 -km default
```

File: keyboards/ymdk/ymdk_np21/readme.md [hunk 7]
```diff
@@ -10,11 +10,11 @@ ps2avrGB based number-pad sold fully assembled by YMDK on Aliexpress.
 
 Make example for this keyboard (after setting up your build environment):
 
-    make ymdk_np21:default
+    make ymdk/ymdk_np21:default
```
Comment (filterpaper) [inline keyboards/ymdk/ymdk_np21/readme.md:13]:
```suggestion
    qmk compile -kb ymdk/ymdk_np21 -km default
```

File: keyboards/ymdk/ymdk_np21/readme.md [hunk 8]
```diff
@@ -10,11 +10,11 @@ ps2avrGB based number-pad sold fully assembled by YMDK on Aliexpress.
...
 
 Flashing example for this keyboard ([after setting up the bootloadHID flashing environment](https://docs.qmk.fm/#/flashing_bootloadhid))
 
-    make ymdk_np21:default:flash
+    make ymdk/ymdk_np21:default:flash
```
Comment (filterpaper) [inline keyboards/ymdk/ymdk_np21/readme.md:17]:
```suggestion
    qmk flash -kb ymdk/ymdk_np21 -km default
```

**Review by filterpaper — CHANGES_REQUESTED** [review]
This is also a good opportunity to update the compilation and flashing commands to `qmk` script that supports more build options and agnostic to current-directory location.

**Comment by ennation** [comment]
> This is also a good opportunity to update the compilation and flashing commands to `qmk` script that supports more build options and agnostic to current-directory location.

Agreed. Still working out some issues with the YMD75 compile though. @drashna said in the QMK Discord that I might need to update some include statements for that board, but after looking at other boards with a similar file hierarchy, I'm not sure what I need to do to them.

**Pull request closed without merging**